documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/CertificationScheme/Certification_Roles_and_Responsibilities.md

```diff
@@ -43,52 +43,68 @@ The IDSA is not actively involved in a participant or core component certificati
 
 Its responsibilities include:
 
--   Formulating and defining the certification scheme in cooperation with the International Data Spaces Association, including the evaluation procedures, participant and core component certification approaches as well as their underlying criteria catalogues.
+-   Formulating and defining the certification scheme in cooperation with the International Data Spaces Association, including the evaluation procedures, operational environment and core component certification approaches as well as their underlying criteria catalogues.
 
-## Evaluation Facility
-
--   Ensuring correct implementation and execution of the Industrial Data Space certification scheme, including the supervision of ongoing evaluations.
+-   To provide and enable the functionality of the checklist approach. (Connector Certification, Trust Level 1, Assureance Level 1).
+-   - For the checklist-approach certification no Evaluation Facility is needed. The Certification Body provides a portal, where applicants can conduct a self-assessment of their connector by answering a questionnaire.
+-   - The answers will be evaluated by the Certification Body directly.  It will inform the applicant about the outcome directly and issue the certificate if the answers have been according to the IDS Certification Scheme. 
 
--   Ensuring continuous adherence to the Industrial Data Space certification scheme following up on changes und updates received from the IDSA.
+-   Approval of the Evaluation Facilites. 
+-   - For the Assurance Level 2, Evaluation Facilites are needed. They evaluate the components in detail and make sure that they act according to the IDS Certification Scheme. 
+-   - Before they start evaluating, the Certification Body ensures, that the evaluation facility's tools and tests are valid. This is beeing done within the approval process. During dedicated workshops the Certification Body examinates the skills of the evaluators. 
 
--   Analyzing existing "base" certificates (e.g. for organizations or for software and hardware security components) to determine their validity and sufficiency, and deciding about their acceptance within the Industrial Data Space certification scheme.
+-   Monitoring the performance of the Evaluation Facilites.
+-   - To ensure highest quality of the certification process the Certification Body participates in the evaluations on a selective basis.
+-   - The Certification Body also reminds the Evaluation Facility to get re-approved after two years if no evaluation has been conducted during that period.
+-   - Whenever new guidelines are developed and published by the association, the Certification Body informs the Evaluation Facilities. 
+-   -   Decision about approval or exclusion of Evaluation Facilities for/from executing IDS evaluations (based on ongoing monitoring and [CRIT-EF]).
```

Comment:
Where shall the reference "CRIT-EF" point to? - if it is a referral to the approval scheme, we could directly add a link to the respective folder here in the repository

Reply:
This content was already in the text when I started, I thought it came from you.

Reply:
I think we had a correct reference in the PDF-Version of the document where we had [CRIT-EF] then defined in the bibliography. We should check there and then adjust it as necessary